### DISCUSSION TITLE
Dumb Win 11 defender / master.zip download

### DISCUSSION THREAD

**Discussion by silviotamaso** [leecher1337/ntvdmx64#243] [Q&A] [answered]
I am sorry to bring this, very sadly for me, the zip download of this awesome piece of soft (no irony at all) didn't work here cause win11 def thinks that's no good. Status bar at the bottom of browser says "ntvdmx64-master.zip failed - virus detected". Thanks. Keep rockin'.

**Comment by emendelson** [comment]
You can easily work around that issue. Search the internet for the answers.

**Comment by leecher1337** [comment, marked as answer]
Disabling the "Safe Browsing" bullsh*t normally helps, i.e. for Chrome: https://www.minitool.com/news/how-to-stop-chrome-from-blocking-downloads.html
